frontend/packages/container-security/src/plugin.ts

```diff
@@ -18,6 +18,9 @@ import { ImageManifestVulnModel } from './models';
 import { ContainerSecurityFlag } from './const';
 import { securityHealthHandler } from './components/summary';
 import { WatchImageVuln } from './types';
+import i18next from 'i18next';
+
+// t('details-page~Vulnerabilities')
```

Comment:
It's generally better to have the comment right by the string where possible so they don't get out of sync.

Reply:
fixed